mlir/test/Target/LLVMIR/Import/ident.ll

```diff
@@ -0,0 +1,6 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
+
+; CHECK: module attributes {
+; CHECK-SAME: "flang version 61.7.4"
```

Comment:
Nit: maybe check that this is indeed imported as `llvm.ident` and not some other string.